### PR TITLE
Update _testembed.c

### PR DESCRIPTION
**Pull request by sidhys**
Changes: Changed data type int to size_t in loops & combined 2 small print statements

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@sidhys1

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by sidhys1** [comment]
I've signed the CLA, waiting for the bot to update

File: Programs/_testembed.c [hunk 1]
```diff
@@ -1182,7 +1182,7 @@ static int test_open_code_hook(void)
     return result;
 }
 
-static int _audit_hook_clear_count = 0;
+static int _audit_hook_clear_count = 0;
```
Comment (arhadthedev) [inline Programs/_testembed.c]: A trailing space is probably added by mistake.

Comment (sidhys) [reply Programs/_testembed.c]: it was indeed added by mistake, i just fixed it!

**Comment by AlexWaygood** [comment]
> I've signed the CLA, waiting for the bot to update

It can take up to two working days to update, the form has to be manually checked by a human :)

I'd also recommend opening an issue at https://bugs.python.org, however, with a description of the problem that this PR solves. Once you've done so, you can change the title of this PR to "bpo-XXXX: improve _testembed.c" (or whatever), and the CI check for a BPO issue will automatically go green.

**Pull request closed without merging**